.eslintignore

```diff
@@ -4,6 +4,6 @@
 /docs/
 /public/
 /packages/*/dist/**/*
-/packages/*/node_modules
+**/node_modules/**/*
```

Comment:
Can we not anchor the directory to the project root and ignore the directory rather than all the contents?

```suggestion
node_modules/
```